src/main.rs

```diff
@@ -1,4 +1,4 @@
-#![warn(missing_docs)]
+//#![warn(missing_docs)]
```

Comment:
Why was this commented out?

Reply:
oh, whoops yeah didn't mean to leave that in